python/tvm/relax/training/loss.py

```diff
@@ -0,0 +1,240 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=redefined-builtin, invalid-name
+"""Loss functions library for relax."""
+
+from typing import Optional, Union
+from tvm import relax
+from ..expr import Expr, Var, Function, StructInfo
+
+from ..op import abs, sum, mean, subtract, multiply
+from ..op.nn import log_softmax, nll_loss
+
+
+__all__ = ["L1Loss", "MSELoss", "CrossEntropyLoss"]
```

Comment:
Seems that we can export Loss, L1Loss, MSELoss, CrossEntropyLoss by default. Do we need  this explicit export?

Reply:
The base class Loss should not be exposed to the external IMO

Reply:
Could we inherit this from outside the class? Say, some user want to define a loss by themselves

Reply:
I think in this case, the user should add his customized loss in the `loss.py` file